### DISCUSSION TITLE
Why this codespace is taking too much time?

### DISCUSSION THREAD

**Discussion by hasnainvakil** [community/community#33416] [Codespaces]
![CS50-Erroe](https://user-images.githubusercontent.com/113837066/190902419-92017e5c-c3f5-4dd1-a26d-d883bdef9875.png)

I have started codespace with code.cs50.io, It's taking a lot of time...
Anyone please help me to fix it...

**Comment by samruddhikhandale** [comment]
Looking at the environment state transition records for `hasnainvakil-code50-113837066-6qxpv4497wjc776`, it appears to me that Codespace was connected several times in a reasonable amount of time (~ 17s) since this discussion was created. 

Let us know if you are seeing this problem again.

**Reply by Mrexplorer429** [reply]
The codespace is taking a lot of time to load for me ( more than 4 mins). Can you help me solve this?
![Visual Studio Code and 2 more pages - Personal - Microsoft​ Edge 6_21_2023 4_13_59 PM](https://github.com/community/community/assets/130761335/b6f3b535-6b2e-44cf-9d5f-2db164ee0399)

**Reply by samruddhikhandale** [reply]
@Mrexplorer429 Can you provide details of your codespace? (eg codespace name or ID)

**Reply by Mrexplorer429** [reply]
I use to use the Website IDE but now, i am using the VS desktop app to Write code. It's faster in loading so I don't need help now. Anyway thank you for trying to solve my problem and I am sorry for making you wait.

**Comment by kanishkakalra** [comment]
why my codespace is taking too much time
![image](https://github.com/community/community/assets/162669030/54186834-63f7-4a88-be4a-f45badb00711)

**Comment by dinith-rusiru** [comment]
slow network

**Reply by dinithrusiru1** [reply]
d

**Comment by kanishkakalra** [comment]
again it is taking too much time and time there is no slow network

**Comment by kanishkakalra** [comment]
![image](https://github.com/community/community/assets/162669030/c89b4f1f-3c4e-48fb-b1a0-770a07afa8c8)
even i downloaded it. 
can you please help @samruddhikhandale

**Reply by samruddhikhandale** [reply]
Are you still facing issues? We were seeing connection problem for two days, I wonder if this is related.

**Comment by sethantanah** [comment]
hmm, I am facing similar issues here, It was working fine while free, I upgraded and now its worse.

**Comment by Tareqkhaled1998** [comment]
![image](https://github.com/community/community/assets/108143455/ebbc51cc-7577-4329-913d-e43cddddb064)
i'm also facing the same issue it is taking ages to load
@samruddhikhandale
